xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -111,12 +107,12 @@ Node getNode() {
     }
 
     @Override
-    CallCredentials getCallCredentials() {
-      return MoreCallCredentials.from(ComputeEngineCredentials.create());
+    List<ChannelCreds> getChannelCredentials() {
+      return Collections.unmodifiableList(channelCredsList);
     }
   }
 
-  private static Bootstrap readConfig() throws IOException {
+  private static BootstrapInfo readConfig() throws IOException {
```

Comment:
It seem this change (introducing `BootstrapInfo`) is not necessary if `Boostrap` has `getServerConfig()` instead of `getServerUri()` and `getChannelCredentials()`.

Reply:
This is private, `BootstrapInfo` better reflects that is a data class.

Reply:
You don't need `BootstrapInfo`. Now your `Bootstrap` getters are inhomogeneous, it has first level item getters (node) and second level item getters (channelCreds and serverUri). When more and more new fields are added to `Bootsrap` in the future, what levels on getters you will have? It should be all first level item getters to be homogeneous.

Reply:
I mis-looked at `Bootstap` as `Boostrapper`. Now I you basically replaced `Bootstrap` with `BootstrapInfo`. Yes, you do need `BoostrapInfo`. 

My comment on `Bootsrapper`'s getters are inhomogeneous is still the valid.

Reply:
Why is that a problem? Users call `bootstrapper.getChannelCredentials()` to get a list of `ChannelCreds` instances and then find one it supports. Then call `creds.getConfig()` to populate information needed. I don't see any problem here.